.travis.yml

```diff
@@ -14,6 +17,11 @@ cache:
     - node_modules
     - packages/*/node_modules
 
+before_install:
+  - nvm install-latest-npm
+install:
+  - if [ $TRAVIS_NODE_VERSION == "12" ]; then nvm install 10 && npm install && nvm use 12; else npm install; fi
```

Comment:
We shouldn't switch Node versions after installing - that would obscure any Node-12-specific installation issues affecting packages other than `@litexa/assets-wav`.

Instead, we could potentially set the `litexa/assets-wav` package as an optional dependency, and then update the automated scripts to use `npm install --no-optional` and `lerna run coverage --ignore @litexa/assets-wav` while testing Node 12.

Reply:
After some more digging, I was able to use `lerna bootstrap --ignore @litexa/assets-wav` instead.

Reply:
Never mind. `--ignore` does not do what I was hoping it would do. It still bootstraps assets-wav and therefore wants to install its lame dependency.